### PR TITLE
feat(developer): add LicenseFile as property of package 🎺

### PR DESCRIPTION
**Pull request by mcdurdin**
Adds license file to .kps, kmp.json, kmc-package, package editor, and new project templates. The intent is for LICENSE.md to be the file used, and for any keyboards on keymanapp/keyboards, it should always be a standard MIT license.

Note: see below for upcoming enhancement to this:
* #9477

# User Testing

TEST_LICENSE_FIELD: In the package editor, add LICENSE.md to the package. Then, in the Details tab, select LICENSE.md in the License File field. Save the package, close and reopen and ensure that the license file field is still correct. Clear the field, save, and verify that on reloading the field is still empty.

### CODE REVIEW

File: common/web/types/src/package/kps-file.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export interface KpsFileOptions {
   followKeyboardVersion?: string;
   readMeFile?: string;
   graphicFile?: string;
+  licenseFile?: string;
```
Comment (darcywong00) [inline common/web/types/src/package/kps-file.ts:44]: Does this get added to https://github.com/keymanapp/api.keyman.com/blob/466072dec5816fefb98ae2df06f0bb6957086eaa/schemas/kps/7.0/kps.xsd#L28-L39 or another schema?

File: common/windows/delphi/packages/PackageInfo.pas [hunk 2]
```diff
@@ -780,6 +805,19 @@ procedure TPackageOptions.SetGraphicFile(const Value: TPackageContentFile);
...
 end;
 
+procedure TPackageOptions.SetLicenseFile(const Value: TPackageContentFile);
+begin
+  if Assigned(FLicenseFile) then FLicenseFile.RemoveNotifyObject(LicenseRemoved);
+  if not Assigned(Value) then
+    FLicenseFile := nil
+  else
+  begin
+    if Value.Package <> Package then raise EPackageInfo.CreateFmt(SGraphicNotOwnedCorrectly, [Value]);
```
Comment (darcywong00) [inline common/windows/delphi/packages/PackageInfo.pas:815]: Is there an error for `SLicenseNotOwnedCorrectly`?

File: common/web/types/src/package/kps-file.ts (hunk 1 above)
Comment (mcdurdin) [reply common/web/types/src/package/kps-file.ts:44]: Yes, copying the schemas over to api.keyman.com will come in a future PR once they have all stabilised. The changes are current in this PR's kps.xsd.

File: common/windows/delphi/packages/PackageInfo.pas (hunk 2 above)
Comment (mcdurdin) [reply common/windows/delphi/packages/PackageInfo.pas:815]: Good catch.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**